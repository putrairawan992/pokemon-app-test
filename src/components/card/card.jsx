import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import MaterialUICard from "@material-ui/core/Card";

const useStyles = makeStyles({
  root: {
    minWidth: 350,
  },
  bullet: {
    display: "inline-block",
    margin: "0 2px",
    transform: "scale(0.8)",
  },
  title: {
    fontSize: 14,
  },
  pos: {
    marginBottom: 12,
  },
});

export const Card = ({ children, ...props }) => {
  const classes = useStyles();

  return (
    <MaterialUICard className={classes.root} {...props}>
      {children}
    </MaterialUICard>
  );
};

export default Card;
