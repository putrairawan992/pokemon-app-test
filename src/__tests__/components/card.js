import React from "react";
import renderer from "react-test-renderer";
import { Card } from "../../components/card/card";

it("Should render card correctly", () => {
  const component = renderer.create(
    <Card>
      <h1>hello world</h1>
    </Card>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
