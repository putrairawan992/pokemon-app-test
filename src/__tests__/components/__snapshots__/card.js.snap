// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render card correctly 1`] = `
<div
  className="MuiPaper-root MuiCard-root makeStyles-root-1 MuiPaper-elevation1 MuiPaper-rounded"
>
  <h1>
    hello world
  </h1>
</div>
`;
